client/components/ReviewHeader.jsx

```diff
@@ -22,12 +22,25 @@ const ReviewHeader = (props) => {
   );
 };
 
+ReviewHeader.defaultProps = {
+  info: {},
+  userPicture: 'http://placehold.it/36x36',
+  user_profile_url: 'http://placehold.it/179x179',
+  username: '',
+  date: '2019-10-05T07:00:00.000Z',
+};
+
 ReviewHeader.propTypes = {
-  info: PropTypes.arrayOf(PropTypes.object).isRequired,
-  userPicture: PropTypes.string.isRequired,
-  user_profile_url: PropTypes.string.isRequired,
-  username: PropTypes.string.isRequired,
-  date: PropTypes.instanceOf(Date).isRequired,
```

Comment:
It was ok for eslint but then when I loaded it in the browser I got some errors because it was set to required but sometimes it's undefined (when no data is pulled in yet) so I had to set up default props instead.